### ISSUE TITLE
Feature Request: View Suricata log

### ISSUE THREAD

**Issue by tpcr** [opnsense/core#2158]
We need a way to view the Suricata log from the WebGUI.

**Comment by fichtner** [comment]
I don't disagree, but may I ask why specifically? :)

**Comment by tpcr** [comment]
To check for errors in rules and broken interface connections. I personally use WinSCP and goto /var/logs/suricata.log. Most other Services have a feature to view the log, but not Suricata, and IMO this one is critical.

**Comment by fichtner** [comment]
I'll get this done shortly, ok.

**Comment by fichtner** [comment]
This should do the trick:

    # opnsense-patch 64ad42d

Cheers,
Franco

**Comment by tpcr** [comment]
Thanks, Great work, Need to add that to next release.

**Comment by fichtner** [comment]
Already asked for an ok, we'll see :)

**Comment by tpcr** [comment]
Upgraded to devel release, then returned to production and re-applied patch. Now have this error when trying to enter log
Parse error: syntax error, unexpected '<', expecting end of file in /usr/local/www/diag_logs_suricata.php on line 9
I manually checked the patches and everything looks fine.

**Comment by fichtner** [comment]
The file is only 8 lines long. opnsense-patch must have appended it the second time you ran it. Just remove the duplicated lines from the file.


Cheers,
Franco

**Comment by tpcr** [comment]
Already did that, It was appended, but I resolved that and the error is still there. Checked all of the patch entries. This happened due to upgrading to the devel version, then reversing that and going back to production. Then I noticed the log option was not there so I applied the patch and that's when the error started. Tried lots of reboots. Guess the next option is to reinstall OPNsense unless you have any suggestions. Maybe uninstall the patch and reapply?
